src/modules/fancyzones/editor/FancyZonesEditor/Models/CanvasLayoutModel.cs

```diff
@@ -56,15 +63,15 @@ public CanvasLayoutModel(string name)
         public void RemoveZoneAt(int index)
         {
             Zones.RemoveAt(index);
-            FirePropertyChanged("Zones");

```

Comment:
Check this https://github.com/microsoft/PowerToys/pull/5196#discussion_r460406102

Reply:
Resolved.